### PR TITLE
Space Travelers' Project

### PR DESCRIPTION
**Pull request by tarek102**
### In this PR:

- Setup project, created the app, installed dependencies like bootstrap, react-bootstrap, redux, redux-toolkit...

- Created the file structure using the Duck method

- Created a service file called missions.js to fetch API data

- The fetched data is then  rendered on the browser

- Dispatched an action to store the selected data in the Redux store

- The list of missions is rendered in the corresponding routes

- React-bootstrap is used to style the UI

### CODE REVIEW

File: src/components/MyProfile.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+                {
+              reservedMissions.map((e) => <MissionList key={e.id} mission_name={e.mission_name} />)
+            }
+              </td>
+            </tr>
+
+          </tbody>
+        </Table>
+      </div>
+    </div>
```
Comment (muneebulrehman) [inline src/components/MyProfile.js:42]: Your layout for `My Profile` section is not matching the provided design template. Kindly update this section accordingly.
| Your design  | Figma design |
| ------------- | ------------- |
| image | image |

File: src/components/NavBar.jsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+import { Navbar, Nav, Container } from 'react-bootstrap';
+import { LinkContainer } from 'react-router-bootstrap';
+
+function NavBar() {
+  return (
+    <Navbar activekey={window.location.pathname} expand="md" sticky="top" className="Navbar bg-light">
+      <Container className="NavContainer">
+        <LinkContainer to="/space_travelers_hub/" rel="noopener noreferrer">
+          <Navbar.Brand>
+            Space Travelers&apos; Hub
```
Comment (muneebulrehman) [inline src/components/NavBar.jsx:11]: You need to use the same logo as in the sample design provided. Kindly update this section and then ask for a re-review.

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi <STUDENT_MENTION>, 

Good job so far! I see you put a lot of effort into this project. However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter errors
- Descriptive PR
- All tests passed
- Good code structure.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by EricMbouwe — APPROVED** [review]
Hi @tarek102, 

Your project is complete! ⚡🔥  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 🎉🔥
- Styles ✔
- Structure ✔
- State management ✔
- Linters ✔

## Optional suggestions ☂
- [ ]  Meanwhile, the changes requested by the previous reviewer are optional I would recommend implementing them.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**